src/objects/Logos.ts

```diff
@@ -0,0 +1,59 @@
+import { Scene } from "phaser";
+import * as CONFIG from "../utils/configuration";
+
+export class Logos {
+    private static instance: Logos | null = null;
+    private scene: Scene;
+    private container: Phaser.GameObjects.Container | null = null;
+    
+    private constructor(scene: Scene) {
+        this.scene = scene;
+        this.create();
+        this.setupSceneTransitions();
+    }
+    
+    public static get(scene: Scene): Logos {
```

Comment:
as said below, i don't think the get is actually needed? Making the constructor public is probably good enough?